### DISCUSSION TITLE
why I just can't open my website deployed in vercel?

### DISCUSSION THREAD

**Discussion by wine-fall** [vercel/community#6833] [Help] [answered]
### Summary

[here](https://vercel.com/winefalls-projects/pwa-with-react/GkW72Enmm156bFbHv6HSiGFcqCkC) is my project dashboard, I don't know if you can open it or not

there is no error in my building log

![image](https://github.com/vercel/community/assets/62830944/67f1aed9-b9de-416a-a63d-895905322522)

and no error in other logs

![image](https://github.com/vercel/community/assets/62830944/ee2b17bc-d6f7-48e0-a385-42041fdc910a)

but when visit this url: [pwa-with-react.vercel.app](https://pwa-with-react.vercel.app/)

it shows:

![image](https://github.com/vercel/community/assets/62830944/febf16e7-3e8e-494a-af5d-6654405c8042)

and the devtool shows:

![image](https://github.com/vercel/community/assets/62830944/95841b0e-3688-40b4-a1f2-c3394e311ddc)

why? Need I pay some money for this?

### Steps to Reproduce

visit this url: [pwa-with-react.vercel.app](https://pwa-with-react.vercel.app/)

**Comment by sahilatahar** [comment]
This site is working in my computer,  follow these steps to fix your problem:

1. **Clear Browser Cache**.
2. **Try a Different Browser**.
3. **Check Internet Connection**.
4. **Disable Browser Extensions**.
5. **Check Firewall/Antivirus Settings**.
6. **Use Incognito/Private Mode**.

**Reply by wine-fall** [reply, marked as answer]
Thank u for ur reply. The reason is just the ip of my location is banned due to some reasons. So just ignore this question. 😁
